examples/imgui_impl_opengl2.cpp

```diff
@@ -127,8 +127,8 @@ static void ImGui_ImplOpenGL2_SetupRenderState(ImDrawData* draw_data, int fb_wid
 void ImGui_ImplOpenGL2_RenderDrawData(ImDrawData* draw_data)
 {
     // Avoid rendering when minimized, scale coordinates for retina displays (screen coordinates != framebuffer coordinates)
-    int fb_width = (int)(draw_data->DisplaySize.x * draw_data->FramebufferScale.x);
-    int fb_height = (int)(draw_data->DisplaySize.y * draw_data->FramebufferScale.y);
+    int fb_width = (int)(draw_data->DisplaySize.x * draw_data->FramebufferScale.x * draw_data->OwnerViewport->DpiScale);
+    int fb_height = (int)(draw_data->DisplaySize.y * draw_data->FramebufferScale.y * draw_data->OwnerViewport->DpiScale);
     if (fb_width == 0 || fb_height == 0)
         return;
```

Comment:
Why isn't `DpiScale` baked into `FramebufferScale` ?

Reply:
`FramebufferScale` is a workaround for MacOS having window size reported to user as if it was 96 DPI while it covers 2x as many pixels. Eventually either `FramebufferScale` or `DpiScale` has to disappear here.